modules/mqtt/destination/mqtt-worker.c

```diff
@@ -28,12 +28,30 @@
 
 #include <stdio.h>
 
+#define TIMEOUT     10000L
 #define DISCONNECT_TIMEOUT 10000
 
-static void
+static LogThreadedResult
 _mqtt_send(MQTTDestinationWorker *self, char *msg)
 {
-  // TODO
+  MQTTClient_message pubmsg = MQTTClient_message_initializer;
+  MQTTClient_deliveryToken token;
+  gint rc;
+  LogThreadedResult result = LTR_SUCCESS;
+
+  pubmsg.payload = msg;
+  pubmsg.payloadlen = (int)strlen(msg);
+  pubmsg.qos = 0;
+  pubmsg.retained = 0;
```

Comment:
[optional]
I could imagine a separate function, which returns an `MQTTClient_message` from `MQTTDestinationWorker *` and `gchar *`.